java-squid/src/test/java/org/sonar/java/ast/visitors/PublicApiCheckerTest.java

```diff
@@ -43,17 +43,20 @@
 
   private PublicApiChecker publicApiChecker;
   private CompilationUnitTree cut;
+  private boolean analyseAccessors;
```

Comment:
It is not always set to true, see line 113. It is changed before re-doing the test.